manifests/base/kustomization.yaml

```diff
@@ -5,7 +5,7 @@ kind: Kustomization
 images:
 - name: quay.io/argoproj/argocd
   newName: quay.io/argoproj/argocd
-  newTag: latest
+  newTag: latest@sha256:f23d3abaa63df1109a86e54a4a14e96d345eec599136e6e1c4573c835a2bba3f
```

Comment:
Eh, I think pinning this in the master branch is a bad idea. We'd have to update it after every commit.